syntax = "proto2";

package xray;


message Vec3{
	required float x = 1;
	required float y = 2;
	required float z = 3;
}

message XRayConfig{
	required string input_path = 1;
	required string output_path = 2;
	
	
}

message PhysicsConfig{

}

message GeneratorConfig{

}

message Config{
	optional XRayConfig xrayconfig = 1;
	optional PhysicsConfig physicsconfig = 2;
	optional GeneratorConfig generatorconfig = 3;	